### DISCUSSION TITLE
Span of days on screen : Lock.

### DISCUSSION THREAD

**Discussion by LAQaiguebelet** [FossifyOrg/General-Discussion#461] [Ideas]
Thanks for having forked and for maintaining this great app.
This is a suggestion from an everyday user.

The slider used to set the day-span is always available on the main screen. It can therefore be manipulated unintentionally during a simple consultation, and undo its preferred setting.

Is there a way to improve the every-day usability, by adding a "Lock" button, or by moving this day-span setting to the settings/parameters page of the app?

Best regards,
Laurent

**Comment by Aga-C** [comment]
There's already `Allow customizing day count` in Settings that does exactly this.
